sdk/entra/WebJobs.Extensions.AuthenticationEvents/CHANGELOG.md

```diff
@@ -0,0 +1,12 @@
+# Release History
```

Comment:
Is "entra" the correct name for the service directory? What does this represent?

Reply:
Had a few discussions with @jsquire and PM (Nick Gomez) and this was the best fit. So we are happy with this.